backend/lambda/lambdaretrievedatalibrary/lambda_return_data.py

```diff
@@ -0,0 +1,119 @@
+import json
+import os
+import boto3
+from botocore.exceptions import ClientError
+import psycopg2
+from psycopg2 import OperationalError, sql
+from credentials import db_creds
+
+def getCredentials(secret_Name) -> db_creds:
```

Comment:
because it's an external function, please change the name to something more meaningful, like `get_postgres_credentials` (BTW, the python naming convention is snake_case) :)